kythe/go/platform/analysis/proxy/proxy.go

```diff
@@ -265,6 +325,26 @@ func decodeEntries(jsonArray json.RawMessage) ([]*spb.Entry, error) {
 	return entries, nil
 }
 
+func decodeWireEntries(jsonArray json.RawMessage) ([]*spb.Entry, error) {
+	var messages []string
+	if err := json.Unmarshal(jsonArray, &messages); err != nil {
+		return nil, err
+	}
+	entries := make([]*spb.Entry, len(messages))
+	for i, msg := range messages {
+		var e spb.Entry
+		b, err := base64.StdEncoding.DecodeString(msg)
+		if err != nil {
+			return nil, err
+		}
+		if err := (proto.UnmarshalOptions{}.Unmarshal(b, &e)); err != nil {
```

Comment:
outer parens are unnecessary

Reply:
Oh, it sees the brace not as a type, but as the opening of the conditional body.

Reply:
Just so. Another way to write it here, without the parentheses, would be:

```go
var opts proto.UnmarshalOptions
if err := opts.Unmarshal(b, &e); err != nil {
```

That also has the advantage of being a little more readable if other options were added later, but of course they are equivalent in effect.